Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Sonata\MediaBundle\Command\RemoveThumbsCommand;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
+use Sonata\MediaBundle\Tests\Entity\Media;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
+
+/**
+ * @author Anton Dyshkant <vyshkant@gmail.com>
+ *
+ * @requires function Symfony\Component\Console\Tester\CommandTester::setInputs
+ */
+final class RemoveThumbsCommandTest extends FilesystemTestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var RemoveThumbsCommand
+     */
+    private $command;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Pool
+     */
+    private $pool;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|MediaManagerInterface
```

Comment:
Should be reversed. MediaManagerInterface is more important to clarify the type than the mock.

Reply:
Plus, the mock uses inheritance, right?